src/main/java/edu/greenblitz/robotName/commands/auto/MoveToPose.java

```diff
@@ -3,18 +3,42 @@
 import com.pathplanner.lib.auto.AutoBuilder;
 import com.pathplanner.lib.path.PathConstraints;
 import edu.greenblitz.robotName.subsystems.swerve.Chassis.ChassisConstants;
+import edu.greenblitz.robotName.utils.GBCommand;
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.wpilibj2.command.Command;
 
-public class MoveToPose {
-    public static Command getPathCommand(Pose2d endPoint) {
-        return getPathCommand(endPoint, ChassisConstants.CONSTRAINTS);
+public class MoveToPose extends GBCommand {
```

Comment:
Pose (stylized as POSE) is an American drama television series about New York City's ball culture, an LGBTQ subculture in the African-American and Latino communities, throughout the 1980s and 1990s. Featured characters are dancers and models, who compete for trophies and recognition in this underground culture and who support one another in a network of chosen families known as Houses.